packages/core/helper-plugin/package.json

```diff
@@ -51,6 +51,7 @@
     "invariant": "^2.2.1",
     "lodash": "4.17.21",
     "match-sorter": "^4.0.2",
+    "mini-css-extract-plugin": "2.4.4",
```

Comment:
do you use this ? it doesn't seem to be loaded somewhere

Reply:
It is loaded in the webpack config of the helper plugin.